test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
They seem not to be sure, plus the spec itself does not say anything. I personally do not mind changing the tests though, as those are arbitrary strings anyways.